src/auto_scheduler/search_policy/sketch_policy_rules.h

```diff
@@ -131,6 +131,29 @@ DEFINE_SKETCH_GENERATION_RULE(RuleCrossThreadReduction);
  * location of the producers of compute ops that perform "fake reduction" with const tensors. */
 DEFINE_SKETCH_GENERATION_RULE(RuleSpecialComputeLocationGPU);
 
+/*! \brief The rule that allows users to generate custom sketches. */
+class RuleCustomSketch : public SketchGenerationRule {
```

Comment:
Should this be named "CustomSketchRule"?

Reply:
Emm... I used this just because our other rule are named "Rulexxxxx". 😄